### ISSUE TITLE
[Feature Request] Default Launch Screen Setting

### ISSUE THREAD

**Issue by Psyclaire** [rileytestut/Delta#355]
Request: The ability to select which emulator loads when the app is launched. (GBA, DS, GBC, etc.)

**Comment by joevegacoding** [comment]
The app launches with the game selection view, and you can swipe between platforms. Is this what you mean by having a default emulator? The app already selects a default platform to choose a game from when it launches; this is the default behavior.

**Comment by Psyclaire** [comment]
> The app launches with the game selection view, and you can swipe between platforms. Is this what you mean by having a default emulator? The app already selects a default platform to choose a game from when it launches; this is the default behavior.

Yes correct, an option to select which emulator is launched when opening the app would be fantastic.
